### ISSUE TITLE
bootstrap-tools needs to be updated

### ISSUE THREAD

**Issue by bgamari** [NixOS/nixpkgs#32782]
## Issue description

In particular the `bash` in http://tarballs.nixos.org/stdenv-linux/x86_64/4907fc9e8d0d82b28b3c56e3a478a2882f1d700f/bootstrap-tools.tar.xz is too old for `make-wrapper`. This has already been worked around in `pkgs/applications/networking/cluster/mesos/default.nix`. This work-around should likely be removed once `bootstrap-tools` has been updated.

**Comment by dezgeg** [comment]
Would also be needed for https://github.com/NixOS/nixpkgs/issues/24954.

**Comment by orivej** [comment]
@bgamari is referring to this commit to mesos: https://github.com/NixOS/nixpkgs/commit/80d1eaa01de9d02e73a99aac15b9be1908839c6a

Only x86_64.nix and i686.nix need updates: ARM bootstrap has new enough bash (4.4-p5).

@edolstra, could you update bootstrap tools? Would you like any help?

**Comment by dtzWill** [comment]
It'd be great if a bootstrap-tools refresh included https://github.com/NixOS/nixpkgs/pull/32954 for compat reasons.

**Comment by vcunat** [comment]
Right, bootstrap currently surely contains glibc < 2.26, so that one should work, and the only problem would be to update it to >= 2.26 _without_ that patch.

**Comment by dtzWill** [comment]
(Yep!)

**Comment by orivej** [comment]
They should be built on `release-17.09` branch, with glibc 2.25.

**Comment by brainrape** [comment]
Is this still an issue?

**Comment by dtzWill** [comment]
AFAIK but I haven't checked recently.  Should be easy to check by anyone at a computer w/Nix-- simplest check should be looking at the glibc version in the bootstrap tools we're using.

Or just git log on the nix expressions that contain the bootstrap urls used :).

**Comment by dtzWill** [comment]
If we're doing this we should probably merge https://github.com/NixOS/nixpkgs/pull/36383 first, to avoid increasing bootstrap tarball size.

----

Looks like last update to boostrap URL's, other than adding boostrap tarballs for musl-native building, was here: 28f87e414186e2f16a094ac354dfe079965e8fe5 -- April 14th 2017.  Since this is well before this issue was created, it does appear this is still an issue.
